### ISSUE TITLE
npm token create error: cannot convert undefined or null to object

### ISSUE THREAD

**Issue by vikas-t** [npm/cli#4672]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

Creating a npm readonly token results in error and exists with exit code 1

```
vikas-t@Vikass-MBP ~ % npm token create --readonly
npm password:
npm ERR! Cannot convert undefined or null to object

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vikas-t/.npm/_logs/2022-04-04T14_55_04_383Z-debug-0.log
vikas-t@Vikass-MBP ~ % echo $?
1
```

### Expected Behavior

The command should not throw any error and should exit with 0

### Steps To Reproduce

1. Run command `npm token create --readonly`
2. Enter password

### Environment

- npm: 8.5.5
- Node.js: v17.8.0
- OS Name: macOs BiSur 11.6.3
- System Model Name: macbook pro m1
- npm config:
```ini
; "user" config from /Users/vikas-t/.npmrc

@npm:registry = "https://npm.pkg.github.com"
//npm.pkg.github.com/:_authToken = (protected)
//registry.npmjs.org/:_authToken = (protected)
admin = ""
developer = ""
fullname = "npm-qa"
npm-torg = "npm-qa"

; node bin location = /Users/vikas-t/.nvm/versions/node/v17.8.0/bin/node
; cwd = /Users/vikas-t
; HOME = /Users/vikas-t
; Run `npm config ls -l` to show all defaults.
```

**Comment by lukekarrys** [comment]
Can you try updating to v8.6.0? I believe this is fixed by 8b65bfd5d610a70e1a860936be1a47f3a3df7f32

**Comment by vikas-t** [comment]
Yep, seems fine.
